RELEASE.md

```diff
@@ -0,0 +1,82 @@
+## How to release
+
+# Before the release
+
+Simple checklist on how to make releases for `tokenizers`.
+
+- Freeze `master` branch.
+- Run all tests (Check CI has properly run)
+- If any significant work, check benchmarks:
+  - `cd tokenizers && cargo bench` (needs to be run on latest release tag to measure difference if it's your first time)
+- Run all `transformers` tests. (`transformers` is a big user of `tokenizers` we need
+  to make sure we don't break it, testing is one way to make sure nothing unforeseen
+  has been done.)
+  - Run all fast tests at the VERY least (not just the tokenization tests).
+- **If any breaking change has been done**, make sure the version can safely be increased for transformers users (`tokenizers` version need to make sure users don't upgrade before `transformers` has). [link](https://github.com/huggingface/transformers/blob/main/setup.py#L154)
+  For instance `tokenizers>=0.10,<0.11` so we can safely upgrade to `0.11` without impacting
+  current users
+- Then start a new PR containing all desired code changes from the following steps.
+- You will `Create release` after the code modifications are on `master`.
+
+# Rust
+
+- `tokenizers` (rust, python & node) versions don't have to be in sync but it's
+  very common to release for all versions at once for new features.
+- Edit `Cargo.toml` to reflect new version
+- Edit `CHANGELOG.md`:
+    - Add relevant PRs that were added (python PRs do not belong for instance).
+    - Add links at the end of the files.
+- Go to [Releases](https://github.com/huggingface/tokenizers/releases)
+- Create new Release:
+    - Mark it as pre-release
+    - Use new version name with a new tag (create on publish) `vX.X.X`.
+    - Copy paste the new part of the `CHANGELOG.md`
+- !! Click on `Publish release`. This will start the whole process of building a uploading
```

Comment:
```suggestion
- ⚠️  Click on `Publish release`. This will start the whole process of building a uploading
```
You can replace "!!" by ⚠️ if you want, I like the visual cue !